### DISCUSSION TITLE
windows without build from source?

### DISCUSSION THREAD

**Discussion by ninetynin** [helix-editor/helix#4827] [Q&A] [answered]
https://docs.helix-editor.com/install.html

I went through this and couldnt find any way to install helix in windows without building from source 
thats why im asking like is there anyway to install without building from source.. 

i dont mind building from source im just curious why u guys didnt kept it thats why im asking nothing else..

**Comment by ninetynin** [comment, marked as answer]
nvm my bad i didnt see about chocolatey

**Reply by kirawi** [reply]
There is also scoop, which is more up-to-date.
